principles/solid/openclosed.md

```diff
@@ -1,18 +1,81 @@
 ## Open/Closed
 
-[TODO]
+An entity of software should be open to extension but closed to modification. We should be able to extend the behavior of our classes without modifying their code.
```

Comment:
The description of all the principles is very poor, it should include the problem, how it solves, and some benefits.